Package.swift

```diff
@@ -331,3 +333,16 @@ if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
         .package(path: "../swift-driver"),
     ]
 }
+
+// Because SwiftSyntax is closely tied to the compiler, only build it if 'SPM_BUILD_PACKAGE_SYNTAX'
+// is set. Package syntax support can only be built alongside a matching SwiftSyntax
+// and compiler toolchain, so only a path-based dependency is supported.
```

Comment:
Since there are runtime requirements that involve embedding `lib_InternalSwiftSyntaxParser.{dylib,so}` into any client that uses this, would it make sense to include a reference to the instructions at `https://github.com/apple/swift-syntax` in this comment?  (although I suppose it's easy enough to find, it might at least call out that there's a runtime requirement here in addition to the build requirement)

Reply:
Good idea. I expanded this comment a bit and added a link to the README

Reply:
Thanks!